From afc5fc671d32cb128ef08bfb8b044b45d653e104 Mon Sep 17 00:00:00 2001
From: Sebastian Kayser <skayser@opencsw.org>
Date: Tue, 20 Jul 2010 01:13:20 +0200
Subject: [PATCH] Make stunnel.conf-sample.in honor --sysconfdir, --localstatedir

In case someone ./configured with a --sysconfdir or --localstatedir
rooted elsewhere than prefix, stunnel.conf-sample.in would have
disregarded it, because it only honors --prefix. This patch
introduces handling for the more specific --sysconfdir/--localstatedir.
---
 tools/stunnel.conf-sample.in |   10 +++++-----
 1 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/tools/stunnel.conf-sample.in b/tools/stunnel.conf-sample.in
index 29acdfc..8791f69 100644
--- a/tools/stunnel.conf-sample.in
+++ b/tools/stunnel.conf-sample.in
@@ -3,14 +3,14 @@
 ; Please make sure you understand them (especially the effect of the chroot jail)
 
 ; Certificate/key is needed in server mode and optional in client mode
-cert = @prefix@/etc/stunnel/mail.pem
-;key = @prefix@/etc/stunnel/mail.pem
+cert = @sysconfdir@/stunnel/mail.pem
+;key = @sysconfdir@/stunnel/mail.pem
 
 ; Protocol version (all, SSLv2, SSLv3, TLSv1)
 sslVersion = SSLv3
 
 ; Some security enhancements for UNIX systems - comment them out on Win32
-chroot = @prefix@/var/lib/stunnel/
+chroot = @localstatedir@/lib/stunnel/
 setuid = nobody
 setgid = @DEFAULT_GROUP@
 ; PID is created inside the chroot jail
@@ -30,12 +30,12 @@ socket = r:TCP_NODELAY=1
 ; CApath is located inside chroot jail
 ;CApath = /certs
 ; It's often easier to use CAfile
-;CAfile = @prefix@/etc/stunnel/certs.pem
+;CAfile = @sysconfdir@/stunnel/certs.pem
 ; Don't forget to c_rehash CRLpath
 ; CRLpath is located inside chroot jail
 ;CRLpath = /crls
 ; Alternatively you can use CRLfile
-;CRLfile = @prefix@/etc/stunnel/crls.pem
+;CRLfile = @sysconfdir@/stunnel/crls.pem
 
 ; Some debugging stuff useful for troubleshooting
 ;debug = 7
-- 
1.7.1

